REQUIRE

```diff
@@ -3,3 +3,4 @@ AbstractFFTs 0.3.0
 Reexport
 Compat 0.62.0
 BinaryProvider 0.3.0
+Conda
```

Comment:
Unfortunate to have to add a dependency on Conda but I don't know of a way around it...

Reply:
Note that the Conda dependency is actually quite small if you aren't using it — it only actually runs the Miniconda installer when you decide to add a Conda package.